index.html

```diff
@@ -183,24 +330,19 @@ <h4>Frameworks</h4>
         <h4>Skills</h4>
         <img src="./images/Union.png" alt="icon">
     </div>
+    <hr>
+</div>
 
 </section>
+<section id="contactlast_section">
```

Comment:
- [ ] Nice job 👍 , Please try to review the **width** in **.last_para**. you can make it between (78% and 100%). It is a suggestion
- [ ] Please Reduce the **margin-left**  to around 34px in your **.last_para**. I am suggesting. 💯 
![r3](https://user-images.githubusercontent.com/63972670/161390319-93bd8e23-24f3-4dcd-ac50-8c7fd3344653.JPG)